src/Uno.UI/Diagnostics/DiagnosticViewManager.TView.TState.cs

```diff
@@ -15,6 +16,7 @@ namespace Uno.Diagnostics.UI;
 internal class DiagnosticViewManager<TView, TState>(Func<IDiagnosticViewContext, TView> factory, Action<IDiagnosticViewContext, TView, TState> update)
 	where TView : FrameworkElement
 {
+	private ConditionalWeakTable<IDiagnosticViewContext, TView> _views = new();
```

Comment:
I agree with Codacy here...
```suggestion
	private readonly ConditionalWeakTable<IDiagnosticViewContext, TView> _views = new();
```